src/wp-content/themes/twentytwelve/functions.php

```diff
@@ -430,7 +441,12 @@ function twentytwelve_content_nav( $html_id ) {
 	 *
 	 * @since Twenty Twelve 1.0
 	 *
-	 * @global WP_Post $post Global post object.
+	 * @global WP_Comment $comment Global comment object.
+	 * @global WP_Post    $post    Global post object.
+	 *
+	 * @param WP_Comment $comment The comment object.
+	 * @param array      $args    An array of comment arguments. @see get_comment_reply_link()
+	 * @param int        $depth   The depth of the comment.
```

Comment:
`twentytwelve_comment()` did not document `$comment` as either a global or one of the parameters